### ISSUE TITLE
Allow user to specify a commit message to be used

### ISSUE THREAD

**Issue by fdellwing** [OWASP/threat-dragon#973]
**Describe what problem your feature request solves**:
"Created by OWASP Threat Dragon" and "Updated by OWASP Threat Dragon" are fine for playing around, but are not enough for a real production use.

**Describe the solution you'd like**:
I would like to be able to write a commit message myself while saving changes in TD. The feature should be disabled by default but has 2 modi if activated via ENV: `enabled` (you can write your own message) and `forced` (you have to write your own message).

**Comment by jgadsden** [comment]
this is a good suggestion @fdellwing , are you able to provide the implementation? I can assign it to you if you like

**Comment by fdellwing** [comment]
I have no idea where I would even start with that. Like, I know where these strings are hardcoded. But I have no idea how to create a popup message field or pass the content around. I'm not a developer and even less a JS developer.
